data/assets/examples/renderable/renderablenodearrow/renderablenodearrow_appearance.asset

```diff
@@ -0,0 +1,38 @@
+-- This example shows an arrow pointing from one scene graph node in the direction of
+-- another. Here, it is created with the Root node as start node and Earth as end node,
+-- but the arrow direction is inverted so that it points towards Root. Some appearance
```

Comment:
Is Root a strange concept to introduce without explanation? Might be easier to go with another planet?

Reply:
Changed it to the Solar System Barycenter instead :)  Solves the rpoblem about visibility